src/Sylius/Behat/Context/Api/Admin/ManagingProductAssociationTypesContext.php

```diff
@@ -285,4 +285,32 @@ public function iRemoveItsNameFromTranslation(string $localeCode): void
             ],
         ]);
     }
+
+    /**
+     * @When I sort the product associations :sortType by code
+     */
+    public function iSortProductAssociationsByCode(string $sortType = 'ascending'): void
+    {
+        $this->client->sort(['code' => self::SORT_TYPES[$sortType]]);
+    }
```

Comment:
`When` methods should be before `Then` ones, and if I see correctly, there is a bit of a mess in this class